pulsar-broker/src/test/java/org/apache/pulsar/client/api/AuthorizationProducerConsumerTest.java

```diff
@@ -209,6 +210,7 @@ public void testSubscriberPermission() throws Exception {
         superAdmin.tenants().createTenant("my-property",
                 new TenantInfoImpl(Sets.newHashSet(tenantRole), Sets.newHashSet("test")));
         superAdmin.namespaces().createNamespace(namespace, Sets.newHashSet("test"));
+        assertNull(superAdmin.namespaces().getPublishRate(namespace));
```

Comment:
According to the [doc](https://github.com/apache/pulsar/blob/38fb839154462fc5c6b0b4293f02762ed4021cd9/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L606), it seems like `-1` represents not configured yet but not `null`.

Reply:
I think this annotation in [doc](https://github.com/apache/pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java#L606) is inaccurate, because `PublishRate` is a composite unit which includes `publishThrottlingRateInMsg` and `publishThrottlingRateInByte`. If `PublishRate` is not configured or deleted then it is `null`, if only one(such as `publishThrottlingRateInMsg`) of them is configured then the default value of the other(`publishThrottlingRateInByte`) is `-1`.
Therefore, maybe we need to correct all rate-related annotation in another PR.

Reply:
Done, PTAL again, thx!

Reply:
@RobertIndie I have corrected all rate-releated annotations, ptal again, thx!